### ISSUE TITLE
Note sure why this error comes up

### ISSUE THREAD

**Issue by resistorsoftware** [kirillplatonov/shopify-hotwire-sample#53]
Nothing happens localhost, but when I deployed this to Heroku, this came up. 

    TypeError (Passed `nil` into T.must):
    sorbet-runtime (0.5.10554) lib/types/_types.rb:218:in `must'
    shopify_api (12.2.1) lib/shopify_api/context.rb:157:in `host_scheme'
    sorbet-runtime (0.5.10554) lib/types/private/methods/call_validation_2_7.rb:57:in `bind_call'
    sorbet-runtime (0.5.10554) lib/types/private/methods/call_validation_2_7.rb:57:in `block in      create_validator_method_fast0'
    shopify_app (21.2.0) lib/shopify_app/controller_concerns/frame_ancestors.rb:11:in `block (3 levels) in <module:FrameAncestors>'
    actionpack (7.0.4) lib/action_dispatch/http/content_security_policy.rb:341:in `instance_exec'
    actionpack (7.0.4) lib/action_dispatch/http/content_security_policy.rb:341:in `resolve_source'
    actionpack (7.0.4) lib/action_dispatch/http/content_security_policy.rb:328:in `block in build_directive'
    actionpack (7.0.4) lib/action_dispatch/http/content_security_policy.rb:328:in `map'
    actionpack (7.0.4) lib/action_dispatch/http/content_security_policy.rb:328:in `build_directive'
    actionpack (7.0.4) lib/action_dispatch/http/content_security_policy.rb:317:in `block in build_directives'
    actionpack (7.0.4) lib/action_dispatch/http/content_security_policy.rb:312:in `each'
    actionpack (7.0.4) lib/action_dispatch/http/content_security_policy.rb:312:in `map'
    actionpack (7.0.4) lib/action_dispatch/http/content_security_policy.rb:312:in `build_directives'
    actionpack (7.0.4) lib/action_dispatch/http/content_security_policy.rb:288:in `build'
    actionpack (7.0.4) lib/action_dispatch/http/content_security_policy.rb:44:in `call'
    rack (2.2.4) lib/rack/session/abstract/id.rb:266:in `context'
    rack (2.2.4) lib/rack/session/abstract/id.rb:260:in `call'

**Comment by kirillplatonov** [comment]
Did you add `HOST` env variable on Heroku?

**Comment by resistorsoftware** [comment]
Nope... what should that be?

**Comment by kirillplatonov** [comment]
App URL: https://github.com/kirillplatonov/shopify-hotwire-sample/blob/main/.env.example
It's a new requirement from ShopifyAPI 10+.

**Comment by resistorsoftware** [comment]
Ok.. I added the https://myapp.herokuapp.com to the HOST of the env list of
things there, and then the App started working.

Interestingly, when it loaded, it did a double-flash of content. I saw the
App load, the home page rendered, then it disappeared, then it rendered
again!

At least it rendered. Thanks for the hot tip there. I will remember to make
a HOST when I upload to the cloud

Dave
